src/js/components/molecules/list-section/list-section.ts

```diff
@@ -102,13 +92,14 @@ export default class ListSection extends Vue implements IListSection {
     handleResize(): void {
         if (!this.showHiddenItems && this.viewportUtil.isMobile) {
             this.showMax = (
-                this.limit && this.limit.smallScreen ? this.limit.smallScreen : this.showItems.smallScreen
+                this.limit && this.limit.smallScreen ? this.limit.smallScreen : undefined
```

Comment:
Does this work with resizing the screen? undefined can result in non reactive data (then you should use null)